cephadm-preflight.yml

```
@@ -48,6 +48,224 @@
     - name: redhat family of OS related tasks
       when: ansible_facts['os_family'] == 'RedHat'
       block:
+        - name: Initialize os_check variable
+          set_fact:
+            os_check: { "failed": false }
+
+        - name: OS Preflight Checks
+          block:
+            - name: Initialize check results
+              set_fact:
+                preflight_results: []
+
+            - name: Ensure OS is RHEL 9+
+              fail:
+                msg: "Ceph requires RHEL 9+. Detected: {{ansible_facts['distribution']}} {{ansible_facts['distribution_version']}}"
```

I believe ansible linter would complain:
```suggestion
                msg: "Ceph requires RHEL 9+. Detected: {{ ansible_facts['distribution'] }} {{ ansible_facts['distribution_version'] }}"
```

---

Addressed in the latest commit.